sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIOTest.java

```diff
@@ -35,7 +35,6 @@
 
 import org.apache.beam.sdk.Pipeline.PipelineExecutionException;
 import org.apache.beam.sdk.coders.Coder;
-import org.apache.beam.sdk.io.bigtable.BigtableIO.BigtableSource;
```

Comment:
are all these changes necessary for some reason? The code seems simpler to me if we import `BigtableSource` rather than type `BigtableIO.BigtableSource` everywhere.